tools/cldr-apps/js/src/cldrVueRouter.js

```diff
@@ -22,20 +26,36 @@ function showPanel(type, el, opts) {
 }
 
 /**
- *
+ * add any Vue components needed here.
+ * @param {App} app
+ */
+function setupComponents(app) {
+  // example:
```

Comment:
I would put the subroutine below its caller

Reply:
done.

I don't need any components at the moment, but I expect to require some soon.

Reply:
added